inflection/src/inflection/dictionary/Inflector.cpp

```diff
@@ -13,7 +13,7 @@
 namespace inflection::dictionary {
 
 const Inflector& Inflector::getInflector(const ::inflection::util::ULocale &locale) {
-    const Inflector* result = npc(npc(DictionaryMetaData::createDictionary(locale))->dictionary)->inflector;
+    const Inflector* result = (npc(DictionaryMetaData::createDictionary(locale))->dictionary)->inflector;
     if (result == nullptr) {
         throw ::inflection::exception::NullPointerException(inflection::util::StringViewUtils::to_u16string("Inflector not found for " + locale.getName()));
     }
```

Comment:
The extra parentheses can be removed.

It's worth noting that the code does try its best to create a stub dictionary, but this null check for the inflector result does trigger when the **required** dictionary was not found. I think the revision is OK, but we have to be extra sensitive here to fail safely instead of crash by derefrencing null.

Reply:
PTAL